examples/other/integration_pytorch.py

```diff
@@ -0,0 +1,538 @@
+
+        SceneHandler._renderer.flush()
+        SceneHandler._renderer.request_draw()
+
+    @staticmethod
+    def start(
+            in_queue: Queue,
+            light_color: gfx.Color = gfx.Color("#ffffff"),
+            background_color: gfx.Color = gfx.Color('#ffffff'),
+            scene_background_top_color: gfx.Color = gfx.Color("#bbbbbb"),
+            scene_background_bottom_color: gfx.Color = gfx.Color("#666666")):
+        border_size = 5.0
+        text_position = np.array([10, 10, 0])
+        text_font_size = 30
+        text_anchor = 'bottom-left'
+
+        SceneHandler._scene_state = SceneState()
+        SceneHandler._in_queue = in_queue
+        SceneHandler._renderer = WgpuRenderer(WgpuCanvas())
+
+        SceneHandler._background_viewport, SceneHandler._background_camera, SceneHandler._background_scene = SceneHandler._create_background_scene(
+            renderer=SceneHandler._renderer,
+            color=background_color)
+
+        SceneHandler._gt_mesh_viewport, SceneHandler._gt_mesh_camera, SceneHandler._gt_mesh_scene = SceneHandler._create_scene(
+            renderer=SceneHandler._renderer,
+            light_color=light_color,
+            background_top_color=scene_background_top_color,
+            background_bottom_color=scene_background_bottom_color)
+
+        SceneHandler._pred_mesh_viewport, SceneHandler._pred_mesh_camera, SceneHandler._pred_mesh_scene = SceneHandler._create_scene(
+            renderer=SceneHandler._renderer,
+            light_color=light_color,
+            background_top_color=scene_background_top_color,
+            background_bottom_color=scene_background_bottom_color)
+
+        SceneHandler._gt_mesh_scene.add(SceneHandler._scene_state.gt_group)
+        SceneHandler._pred_mesh_scene.add(SceneHandler._scene_state.pred_group)
+
+        SceneHandler._gt_mesh_text_scene, SceneHandler._gt_mesh_text_camera = SceneHandler._create_text_scene(
+            text_string='Ground Truth',
+            position=text_position,
+            anchor=text_anchor,
+            font_size=text_font_size)
+
+        SceneHandler._pred_mesh_text_scene, SceneHandler._pred_mesh_text_camera = SceneHandler._create_text_scene(
+            text_string='Prediction',
+            position=text_position,
+            anchor=text_anchor,
+            font_size=text_font_size)
+
+        @SceneHandler._renderer.add_event_handler("resize")
+        def on_resize(event: Optional[gfx.WindowEvent] = None):
+            w, h = SceneHandler._renderer.logical_size
+            w2, h2 = w / 2, h / 2
+            SceneHandler._gt_mesh_viewport.rect = 0, 0, w2 - border_size, h
+            SceneHandler._pred_mesh_viewport.rect = w2 + border_size, 0, w2 - border_size, h
+
+        on_resize()
+        SceneHandler._renderer.request_draw(SceneHandler._animate)
+        run()
+
+
+class PyGfxCallback(Callback):
```

Comment:
docstring for this class and the methods

Reply:
docstring added